README.md

```diff
@@ -1,5 +1,5 @@
 # Eclipse Che documentation project
-[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://che.openshift.io/f?url=https://github.com/eclipse/che-docs)
+[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://che.openshift.io/#https://github.com/eclipse/che-docs)
```

Comment:
Is the # a new way to load a factory?

Reply:
yes https://github.com/eclipse/che/issues/18165

Reply:
but note that it's not supported in CRW yet

Reply:
Yes, we will make sure the content of this PR doesn't go into the 7.26.x branch that will be used for CRW 2.7 docs.